### PR TITLE
feat(color): add FORCE_COLOR environment variable to support chalk

### LINKED ISSUE #86: Colors from child scripts not being used in logged output
Thanks for this tool! I think it's great! I'm not getting colorized output when running concurrently on a script that is outputing stuff with chalk. Here's a repo that reproduces the issue: https://github.com/kentcdodds/chalk-concurrently-bug

output

It's pretty simple. `color-red.js` is:

```javascript
#!/usr/bin/env node
const chalk = require('chalk')

console.log(chalk.red(process.argv.slice(2).join(' ')))
```

Any ideas? This is sort of a blocker for [`p-s`](https://github.com/kentcdodds/p-s) dropping the `--parallel` option and recommending people just use `concurrently`. ([more info](https://github.com/kentcdodds/p-s/pull/94)).

### PR DESCRIPTION
**Pull request by kentcdodds**
Closes #86

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -185,6 +186,9 @@ function run(commands) {
         if (IS_WINDOWS) {
             spawnOpts.detached = false;
         }
+        if (supportsColor) {
```
Comment (kentcdodds) [inline src/main.js]: I opted for this if statement rather than what we talked about in the issue because it makes the intent a little more clear.

**Comment by kentcdodds** [comment]
You can use squash and merge feature and change the commit message yourself right in the github ui: https://github.com/blog/2141-squash-your-commits

**Comment by kentcdodds** [comment]
Updated. I think this is correct...

**Comment by boneskull** [comment]
Ha, I just ran into a corner case here, where we're launching Mocha as a subprocess and assertion it's not dumping colors into a non-tty session.  I had to explicitly unset `FORCE_COLOR`.

Is it not possible to consume the `--color` flag that `supports-color` detects?  Granted, I don't love the idea of a library looking for a flag in this manner...

**Pull request merged**